### ISSUE TITLE
Events page

### ISSUE THREAD

**Issue by shuchibooksmith** [sonnetmedia/brooklinebooksmith.com#11]
So here are two websites that I've referred to when thinking about the style of the events page:

http://www.riverrunbookstore.com/events
what works on their events page
-each event is separated out by a line. I'd love something similar for our events page - whether it be a line, a color block, whatever. just something that visually separates each event from one another. i'll leave that up to you.
-the calendar icon on the right, which gives an easy visual for people. they can look at the cover and then pivot right to the calendar page. but instead of it just be a calendar page that doesn't link to anything, it'd be great if they could click on it and have it take them to a google calendar.  (though this would directly mimic Harvard Book Store's website, so we'd have to make sure it didn't look exactly the same)
-the basic information included at the top. if you take our grey sidebar right now, you can pull the last paragraph about ordering signed copies of the book, and  i can maybe give you some separate text for the introductory portion, something incredibly brief.

I also referred to Harvard Book Store's events page. In general I think it's a bit of a mess, and it takes forever to load. BUT I really like how they utilize the tab format that I discussed in my last round of notes to you.
as an example:
http://www.harvard.com/event/chelsea_clinton/
-For our two tab format, I'd like one to say Book and one to say Event. The Book tab would include the book summary, maybe author info. The Event tab would include the necessary information (whether the event is in store, handicap accessibility, ticket details, etc.)
-I also like that "Ticketed" banner. We need something similar for our ticketed events.

As far as how basic information is conveyed, we want to stick as closely to the format we have on our website as possible.
so, on each of their own lines:
day of the week, month, date, time (we didn't like the @ sign)
author name
book title (bold and italicized always)
genre of book
then the two tabs one with Book, the author with Event
for ticketed events, some icon that alerts customers to the fact that the event they're look at is ticketed

-We like the social media options for each event! And are also hoping that the book cover will link directly to the e-commerce side, so someone could just click on the book if they want to order it.
-And on the mobile events page, would it possible to show the next five events, and then you can swipe to the right to show the next five?

**Comment by budparr** [comment]
reiterating these so I can find them:
so, on each of their own lines:
- day of the week, month, date, time (we didn't like the @ sign)
- author name
- book title (bold and italicized always)
- genre of book
- then the two tabs one with Book, the author with Event
- for ticketed events, some icon that alerts customers to the fact that the event they're look at is ticketed

**Comment by shuchibooksmith** [comment]
Yes, exactly! 
I'm also going to attach our november events calendar. This is our wall calendar, it goes up behind the register. (it's also incorrect at the moment, the dates are not on there correctly, but just so you get a general sense of our layout. on the wall calendar we bold the author's names as opposed to book title)

![bsm_nov_progress01_72dpi](https://cloud.githubusercontent.com/assets/15218891/10671093/d18e5ffc-78b6-11e5-8a49-7519188e6ad8.jpg)

**Comment by budparr** [comment]
Thanks for sharing that. I've made some good progress on the events display, and we can talk about it again after Monday's version.

**Comment by shuchibooksmith** [comment]
Add general info (free and open to the public, wheelchair accessibility, etc.) to the righthand side of the events page.

**Comment by shuchibooksmith** [comment]
For general events info, only some Brookline parking meters run until 8 pm. Some go until 6, some until 8.  Saying they all go until 8 is not actually accurate, so you can amend the phrase however you like.

**Comment by budparr** [comment]
@shuchibooksmith I've made this copy: 
> Please note that Brookline parking meters run **as late as 8pm** Monday through Saturday.

I think that captures what you want to get across, that visitors may need to watch their meters possibly as late as eight. Feel free to change though.


Also note that I removed the sharing buttons on individual events in favor of the "Add to calendar" button, which I think is the highest value activity for your users.
